variant_test.go

```diff
@@ -76,3 +77,32 @@ func TestParseVariant(t *testing.T) {
 		}
 	}
 }
+
+var variantJsonTests = []struct {
```

Comment:
...and acronyms should always have a consistent case. In this case, `variantJSONTests`